ppb/engine.py

```diff
@@ -7,15 +7,15 @@
 
 class GameEngine(Engine):
 
-    def __init__(self, first_scene: Type, **kwargs):
+    def __init__(self, first_scene: Type, *, delta_time=None, resolution=None, flags=None, depth=None, log_level=None):
         super(GameEngine, self).__init__()
 
         # Engine Configuration
-        self.delta_time = kwargs.get("delta_time", 0.016)
-        self.resolution = kwargs.get("resolution", (600, 400))
-        self.flags = kwargs.get("flags", 0)
-        self.depth = kwargs.get("depth", 0)
-        self.log_level = kwargs.get("log_level", logging.WARNING)
+        self.delta_time = delta_time or 0.016
```

Comment:
There's no reason to use the None sentinel in any of these cases: None of these defaults are mutable and it'll read more cleanly if they're in the signature.

Please switch the defaults from `None` to the respective values.